boost-maven/boost-boms/boost-mp14-apis-bom/pom.xml

```diff
@@ -32,11 +32,21 @@
 
 	<dependencyManagement>
 		<dependencies>
+			<!--
```

Comment:
Why did you comment this out as opposed to just deleting it?  Had we talked about being unsure here?

Reply:
deleted commented out stanzas